components/BottomNavigation/src/MDCBottomNavigationBar.h

```diff
@@ -164,6 +164,11 @@ typedef NS_ENUM(NSInteger, MDCBottomNavigationBarAlignment) {
  */
 @property(nonatomic, assign) CGFloat itemsContentHorizontalMargin;
 
+/**
+ The amount of horizontal padding on the leading/trailing edges of each bar item. Defaults to 12.
+ */
+@property(nonatomic, assign) CGFloat itemsHorizontalPadding;
```

Comment:
@randallli , I'm wondering if `itemsHorizontalInset` might be a better name?

Reply:
Insets brings to mind https://developer.apple.com/documentation/uikit/uiedgeinsets
so probably not.